src/client/reducers/index.js

```diff
@@ -1,9 +1,11 @@
 import { combineReducers } from 'redux';
 import createNewSession from './sessionReducer';
+import fetchSession from './fetchedSessionReducer';
 import updateProfile from './updateProfile';
 
 const rootReducer = combineReducers({
     createNewSession,
+    fetchSession,
```

Comment:
misnomer here, the one above should be `sessionReducer` instead of `createNewSession`.

Instead of creating a new reducer for every interaction with the session, let's just add actions to the `sessionReducer`. Keeps code cleaner and syncs our stores for all the sessions

Reply:
Check slack, this was actually done by Edvin for his join page. I saw that there were two reducers, and just picked his cuz it looked like there was a little more done with it at this point.

Reply:
Oh this includes code from Edvin's branch that hasn't been merged yet. 

Edvin would still need to make some changes before his code can be merged in, including the changes to the reducers (the store also doesn't have a field for imageURL for the sessions). I left some comments on that PR.

Let's give him a bit to make the changes and then get that merged before we merge this one in.

Reply:
Yea I noticed the discrepancy with the names sessionReducer vs createNewSession, I ended up sticking with the name createNewSession which we can rename. Reducers are now merged to project_2 branch